dracut/modules.d/59kiwi-lib/kiwi-partitions-lib.sh

```diff
@@ -213,6 +213,51 @@ function get_partition_node_name {
     return 1
 }
 
+function get_partition_name {
+    local disk=$1
+    local part_id=$2
+    local part_name
+    local table_type
+    table_type=$(get_partition_table_type "${disk}")
+    if [ "${table_type}" = "gpt" ];then
+        part_name=$(sfdisk --part-label "${disk}" "${part_id}")
+    else
+        part_name=unsupported
+    fi
+    echo "${part_name}"
+}
+
+function recreate_last_partition {
```

Comment:
I think this method is not longer used and could be removed. I really like the last changes you did, so it reuses the already existing partitioning logic.